pkg/provider/kubernetes/crd/traefikio/v1alpha1/serverstransport.go

```diff
@@ -31,7 +31,11 @@ type ServersTransportSpec struct {
 	ServerName string `json:"serverName,omitempty"`
 	// InsecureSkipVerify disables SSL certificate verification.
 	InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty"`
-	// RootCAsSecrets defines a list of CA secret used to validate self-signed certificate.
+	// RootCA gives you the option to use secrets or config maps to store CAs
+	RootCA []RootCA `json:"rootCA,omitempty"`
```

Comment:
```suggestion
	RootCAs []RootCA `json:"rootCAs,omitempty"`
```

Reply:
The slice itself should keep the plural form, to be consistent with the naming of other options.

Reply:
but @sdelicata said singular is better here https://github.com/traefik/traefik/pull/11475#discussion_r1975108077

Reply:
Yep he said that but I think it was only about the RootCA struct, not the slice of RootCA.

Reply:
oh okay, I will refactor